napalm_logs/config/iosxr/ISIS_NEIGHBOR_DOWN.yml

```diff
@@ -0,0 +1,19 @@
+messages:
+  # 'error' should be unique and vendor agnostic. Currently we are using the JUNOS syslog message name as the canonical name.
+  # This may change if we are able to find a more well defined naming system.
+  - error: ISIS_NEIGHBOR_DOWN
+    tag: ROUTING-ISIS-5-ADJCHANGE
+    values:
+      neighbor: ([^ ]+)
+      interface: ([^ ]+)
+      level: (L\d)
+      downReason: (.*)
+    line: 'Adjacency to {neighbor} ({interface}) ({level}) Down, {downReason}'
+    model: NOMODEL
+    mapping:
+      variables:
+        isis/neighbor: neighbor
```

Comment:
The hierarchy defined here under `variables` will reflect the structure of the document generated. The `openconfig-isis` YANG module is derived from the same `openconfig-network-instances`, as well as `openconfig-ospf`. Can you have a look at the similar model here: https://github.com/napalm-automation/napalm-logs/blob/master/napalm_logs/config/junos/OSPF_NEIGHBOR_DOWN.yml#L17?